app/models/nonprofit.rb

```diff
@@ -55,6 +55,7 @@ class Nonprofit < ApplicationRecord
   has_many :payouts
   has_many :charges
   has_many :refunds, through: :charges
+  has_many :disputes, through: :charges
```

Comment:
As an aside, should these has many relationships have their ‘dependent’ status defined? Nullable, destroys, etc?